File: Proyecto/App/models/envio.js
const direccion = require("./direccion");
const persona = require("./persona");
const producto = require("./producto");

const envio = {
    id = "",
    origen:direccion = "",
    destino:direccion = "",
    contacto:persona = "",
    producto:producto = "",
    estado = "",
    fecha_recepcion = "",
    fecha_entrega = ""

};
module.exports = envio;

File: Proyecto/operadorLogisticoAPI/src/operadorLogisticoAPI/Models/envio.cs
namespace operadorLogisticoAPI.Models 
{

    public class Envio
   {
       public long Id { get; set; }
       public Direccion origen { get; set; }
       public Direccion destino { get; set; }
       public Direccion contacto { get; set; }
       public string estado { get; set; }
       public string fecha_recepcion { get; set; }
       public Repartidor repartidor { get; set; }
   }

}